frontend/demo/component/gridpro/grid-pro-edit-column.ts

```diff
@@ -0,0 +1,49 @@
+import '../../init'; // hidden-full-source-line
+import '@vaadin/flow-frontend/gridProConnector.js'; // hidden-full-source-line
+import '@vaadin/flow-frontend/gridConnector.js'; // hidden-full-source-line
+
+import { html, LitElement, internalProperty, customElement } from 'lit-element';
+import '@vaadin/vaadin-grid-pro/vaadin-grid-pro';
+import '@vaadin/vaadin-grid-pro/vaadin-grid-pro-edit-column';
+import '@vaadin/vaadin-grid/vaadin-grid-column';
+import { getPeople } from '../../domain/DataService';
+import Person from '../../../generated/com/vaadin/demo/domain/Person';
+import { applyTheme } from 'themes/theme-generated.js';
+import { GridItemModel } from '@vaadin/vaadin-grid';
+import { GridColumnElement } from '@vaadin/vaadin-grid/vaadin-grid-column';
+
+@customElement('grid-pro-edit-column')
+export class Example extends LitElement {
+  constructor() {
+    super();
+    // Apply custom theme (only supported if your app uses one)
+    applyTheme(this.shadowRoot);
+  }
+
+  @internalProperty()
+  private items: Person[] = [];
+
+  async firstUpdated() {
+    this.items = await getPeople();
+  }
+
+  render() {
+    return html`
+      <!-- tag::snippet[] -->
+      <vaadin-grid-pro .items=${this.items} enter-next-row>
+        <vaadin-grid-column
+          header="Name (read-only)"
+          .renderer=${(root: HTMLElement, _column?: GridColumnElement, model?: GridItemModel) => {
+            const item = model?.item as Person;
+            root.textContent = `${item.firstName} ${item.lastName}`;
+          }}
```

Comment:
The column and model parameters shouldn't need to be optional. The optional `model`'s `item` is cast to `Person` and accessed later, so optional doesn't work here.
```suggestion
          .renderer=${(root: HTMLElement, _column: GridColumnElement, model: GridItemModel) => {
            const { firstName, lastName } = model.item as Person;
            render(
              html`
                ${firstName} ${lastName}
              `,
              root
            );
          }}
```

The renderers in `grid-pro-editors.ts` have the same issue.

Reply:
Well, interestingly `GridBodyRenderer` defines them as optional...not sure why.

Reply:
This is still an issue:
```js
const item = model?.item as Person; // Casting a possible undefined as a Person
item.firstName; // Accessing a property of the possibly undefined object
```

Reply:
Added checks for undefined.